gradoop-examples/src/main/java/org/gradoop/examples/benchmark/grouping/GroupingBenchmark.java

```diff
@@ -76,17 +76,37 @@
    */
   public static final String OPTION_CSV_PATH = "csv";
   /**
-   * Use aggregator (min, max, count, none)
+   * Used vertex aggregator (min, max, count, none)
    */
-  private static final String OPTION_AGGREGATION = "agg";
+  private static final String OPTION_VERTEX_AGGREGATION = "vagg";
```

Comment:
rename this to `OPTION_VERTEX_AGGREGATION_FUNCS` (also for edges) and document accordingly